### ISSUE TITLE
Documentation: Comment in default configuration.nix uses "nix search" incorrectly

### ISSUE THREAD

**Issue by timgott** [NixOS/nixpkgs#221340]
## Problem

The default configuration.nix generated by the NixOS installer contains the comment

```
  # List packages installed in system profile. To search, run:
  # $ nix search wget
```

Seems to be generated from here https://github.com/NixOS/nixpkgs/blob/367bf103d0c16c27c2fd09bc030e829854bd61e5/nixos/modules/installer/tools/tools.nix#L188

However,
1. that command is experimental and not enabled by default
2. once enabled it still does not work, gives the error `error: cannot find flake 'flake:wget' in the flake registries`

It can confuse new users.

## Proposal

The correct command is now `nix search nixpkgs wget`.
Maybe add a link to documentation instead especially while it is still experimental.

## Checklist

- [x] checked [open documentation issues] for possible duplicates
- [x] checked [open documentation pull requests] for possible solutions

[latest Nixpkgs manual]: https://nixos.org/manual/nixpkgs/unstable/
[latest NixOS manual]: https://nixos.org/manual/nixos/unstable/
[nixpkgs-source]: https://github.com/NixOS/nixpkgs/tree/master/doc
[nixos-source]: https://github.com/NixOS/nixpkgs/tree/master/nixos/doc/manual
[open documentation issues]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+label%3A%229.needs%3A+documentation%22
[open documentation pull requests]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+is%3Apr+label%3A%228.has%3A+documentation%22%2C%226.topic%3A+documentation%22

**Comment by JoNilsson** [comment]
@timgott 
Yep..  it's wholly confusing..  but through persistent trial and error i found the way,

```sh
$  nix search nixpkgs wget --extra-experimental-features flakes --extra-experimental-features nix-command
```

**Comment by mschwaig** [comment]
I opened a PR to fix this, because I saw people run into it on social media: https://bsky.app/profile/racc.at/post/3lh3isjazvk2a
